source/actors/player/PlayerHealth.hx

```diff
@@ -37,14 +37,17 @@ class PlayerHealth extends FlxTypedGroup<HeartContainer> {
     }
 
     public function updateGraphics(currentHealth:Float):Void {
-        for (val in 0...countLiving()) {
+        for (val in 0...length) {
+            trace("Health at call " + (val + 1) + ": " + currentHealth);
```

Comment:
Let's strip out this logging line before merging
```suggestion
```

Reply:
I got that line removed